core/src/main/scala/org/apache/spark/storage/BlockManagerMaster.scala

```diff
@@ -195,8 +201,13 @@ class BlockManagerMaster(
         log"${MDC(ERROR, e.getMessage)}", e)
     )(ThreadUtils.sameThread)
     if (blocking) {
-      // the underlying Futures will timeout anyway, so it's safe to use infinite timeout here
```

Comment:
so this comment is not accurate?

Reply:
It will be helpful if the comment is replaced (instead of removing incorrect one) with the explanation why infinite timeout can't be used here.